src/tools/ceph_objectstore_tool.cc

```diff
@@ -1804,33 +1635,21 @@ int ObjectStoreTool::do_import(ObjectStore *store, OSDSuperblock& sb,
         return 11;  // Positive return means exit status
   }
 
-  // Don't import if pool no longer exists
+  // we need the latest OSDMap to check for collisions
   OSDMap curmap;
   bufferlist bl;
   ret = get_osdmap(store, sb.current_epoch, curmap, bl);
   if (ret) {
-    cerr << "Can't find local OSDMap" << std::endl;
+    cerr << "Can't find latest local OSDMap " << sb.current_epoch << std::endl;
     return ret;
   }
-  if (!curmap.have_pg_pool(pgid.pgid.m_pool)) {
-    cerr << "Pool " << pgid.pgid.m_pool << " no longer exists" << std::endl;
-    // Special exit code for this error, used by test code
-    return 10;  // Positive return means exit status
```

Comment:
@liewegas The thrasher code looks specifically for exit status 10, prints a message and ignores the error.  More to the point why would we want to allow the import of a pg for a pool that has been removed?